airtable/airtable.js

```diff
@@ -10,20 +10,21 @@
 */
 import Airtable from 'airtable';
 import { Columns } from './schema';
-import Constants from '../Constants';
 
 require('dotenv-safe').config();
 
-const apiKey = process.env.REACT_APP_AIRTABLE_API_KEY;
+const BASE_ID = process.env.REACT_APP_AIRTABLE_BASE_ID;
 
-require('dotenv-safe').config();
+const API_KEY = process.env.REACT_APP_AIRTABLE_API_KEY;
+const ENDPOINT_URL = 'https://api.airtable.com';
+const VIEW = 'Grid view';
 
 Airtable.configure({
-  endpointUrl: Constants.ENDPOINT_URL,
-  apiKey
+  endpointUrl: ENDPOINT_URL,
+  apiKey: API_KEY
```

Comment:
nit: i had opted for a string in the past because i thought that made it easier to see at first glance that we are using Airlock (with Airtable-esque API), but whatever works

Reply:
yeah i mean for the front end this is the string “airlock” but the backend is not using airlock so it needs an actual API KEY